app/src/main/java/com/testtask/apptesttask/ui/main/MainFragment.kt

```diff
@@ -0,0 +1,112 @@
+package com.testtask.apptesttask.ui.main
+
+import android.os.Bundle
+import android.support.design.widget.BottomNavigationView
+import android.support.v4.app.Fragment
+import com.arellomobile.mvp.MvpView
+import com.arellomobile.mvp.presenter.InjectPresenter
+import com.testtask.apptesttask.R
+import com.testtask.apptesttask.presentation.main.MainPresenter
+import com.testtask.apptesttask.ui.about_me.AboutMeFragment
+import com.testtask.apptesttask.ui.characters.CharactersFragment
+import com.testtask.apptesttask.ui.global.BaseFragment
+import com.testtask.apptesttask.ui.likecharacters.LikeCharactersFragment
+import kotlinx.android.synthetic.main.fragment_main.*
+
+class MainFragment : BaseFragment(), MvpView{
+
+    override var layoutRes: Int = R.layout.fragment_main
+
+    private var containRes: Int = R.id.mainContainer
+
+    private var currentTabTag: String = CHARACTERS
+
+    @InjectPresenter
+    lateinit var presenter: MainPresenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        if (savedInstanceState == null) {
+            childFragmentManager.beginTransaction()
+                    .apply {
+                        add(containRes, createFragmentByTag(ABOUT), ABOUT)
+                        add(containRes, createFragmentByTag(LIKECHARACTERS), LIKECHARACTERS)
+                        add(containRes, createFragmentByTag(CHARACTERS), CHARACTERS)
+                    }.commitNow()
```

Comment:
Also, you must check, when the `Fragment` is restored and `show/hide` values is restored too?